source/docs/advanced/production-ready/media-middleware.md

```diff
@@ -181,7 +181,7 @@ var { getSizesStats, getImage, resetIsMyImage, clearSizesStats } = ((
   return {
     getSizesStats: () => {
       stats.sort(([winSize1], [winSize2]) => winSize1 - winSize2);
-      return stats.map((itm) => itm.join("\t")).join("\n");
+      return console.log(stats.map((itm) => itm.join("\t")).join("\n"));
```

Comment:
This avoids inlined logging

#### Before:
![Google Chrome - New Products 2022-05-04 at 13 10 20](https://user-images.githubusercontent.com/39598117/166670594-3617a818-26ed-48e5-a833-42d38c168696.png)

#### After
![Google Chrome - New Products 2022-05-04 at 13 11 03](https://user-images.githubusercontent.com/39598117/166670660-32477122-bb01-4b8e-908a-847cc1186fc5.png)